bedrock/products/templates/products/landing.html

```diff
@@ -15,136 +16,214 @@
-          <![endif]-->
-          <!--[if !IE]><!-->
-            {# Download link should be locale neutral see issue 7982 #}
-            <li class="mzp-c-menu-list-item menu-desktop"><a href="/firefox/download/thanks/" data-link-type="Desktop" data-download-os="Desktop" data-download-version="standard">{{ ftl('firefox-products-desktop') }}</a></li>
-          <!--<![endif]-->
-          <li class="mzp-c-menu-list-item menu-android"><a href="{{ android_url }}" rel="external noopener" class="ga-product-download" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">{{ ftl('firefox-products-android') }}</a></li>
-          <li class="mzp-c-menu-list-item menu-ios"><a href="{{ ios_url }}" rel="external noopener" class="ga-product-download" data-link-type="download" data-display-name="iOS" data-download-version="ios" data-download-os="iOS">{{ ftl('firefox-products-ios') }}</a></li>
-        </ul>
-      </div>
-      <p><a class="mzp-c-cta-link" href="{{ url('firefox') }}" data-cta-type="link" data-cta-text="See all browsers">{{ ftl('firefox-products-see-all-browsers') }}</a></p>
-    </div>
-
-  {% if ftl_has_messages('firefox-products-surf-stream-and-get-work-done') %}
-    <div class="c-landing-grid-item">
-      <img src="{{ static('img/products/landing/vpn.svg') }}" class="c-landing-grid-img" alt="" width="350" height="247">
-      <h2 class="c-landing-grid-title"><a href="{{ url('products.vpn.landing') }}" data-cta-type="link" data-cta-text="Mozilla VPN">{{ ftl('firefox-products-mozilla-vpn') }}</a></h2>
+      <div class="show-else">{{ download_firefox_thanks(button_class='mzp-t-lg') }}</div>
+      <p class="show-android">{{ google_play_button(href=fx_android_url, id='playStoreLink-firefox') }}</p>
+      <p class="show-ios">{{ apple_app_store_button(href=fx_ios_url, id='appStoreLink-firefox') }}</p>
+    {% endcall %}
+
+    <!-- Focus -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='protocol/img/logos/firefox/browser/focus/logo.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '64',
+          'height': '64',
+          'loading': 'lazy'
+        }
+      ),
+    ) %}
+      <h3 class="mzp-c-picto-heading"><a href="{{ url('firefox.browsers.mobile.focus') }}" data-cta-type="link" data-cta-text="Focus" data-cta-position="heading">{{ ftl('firefox-products-firefox-focus') }}</a></h3>
+      <p>{{ ftl('firefox-products-your-dedicated-privacy') }}</p>
+      <p>
+        <span class="hide-ios">{{ google_play_button(href=fc_android_url, id='playStoreLink-focus') }}</span>
+        <span class="hide-android">{{ apple_app_store_button(href=fc_ios_url, id='appStoreLink-focus') }}</span>
+      </p>
+    {% endcall %}
+
+    <!-- Relay -->
+    {% call picto(
+      body=True,
+      image=resp_img(
+        url='img/logos/relay/logo.svg',
```

Comment:
Only the new `relay/logo.svg` is being used, but this PR also touches the [`relay/logo-firefox-relay.svg`](https://github.com/mozilla/bedrock/pull/14407/files#diff-5378621ba6f8074ad1222684c26c3ece58cf6d614955c69a7898b955b08a8e6e) that's maybe just a mistake that shouldn't be committed?

![Screenshot 2024-04-04 at 15 38 05](https://github.com/mozilla/bedrock/assets/1784648/2e0a175a-e235-4bcd-9a1e-d8e205fc3579)

Reply:
+1 here -- good eye! This should be deleted if not used 👍🏼

Reply:
Right! The logo was added temporarily to override https://github.com/mozilla/protocol-assets/issues/87, but it was only used before https://github.com/mozilla/bedrock/pull/13871 so yes, that can be removed as it is unused now. Or it can be kept in its current location to be handy until it's available in Protocol. (Just probably not this half-empty version;D)

Reply:
I probably hit save after starting to edit it but before renaming in file 🤦‍♀️ Thanks for the catch.

Reply:
Exactly my thoughts;)

BTW the logo version of Relay added / used is actually already in Protocol if you want to simplify: [logos/firefox/relay/logo-white.svg](https://github.com/mozilla/protocol-assets/blob/main/logos/firefox/relay/logo-white.svg)